community/cypher/frontend-3.0/src/main/scala/org/neo4j/cypher/internal/frontend/v3_0/symbols/TypeSpec.scala

```diff
@@ -36,6 +36,7 @@ object TypeSpec {
     CTNumber,
     CTPath,
     CTRelationship,
+    CTGeometry,
```

Comment:
Perhaps call this `CTPoint`? We're likely to want type differentiation between Points and other types of Geometries, and it'll be easier to introduce those if, for now, we just focus on Point.